tests/integration/ha/test_storage_reuse.py

```diff
@@ -79,10 +81,15 @@ async def test_attach_storage_after_scale_down(ops_test: OpsTest) -> None:
     )
 
     # add unit with previous storage attached
-    add_unit_cmd = f"add-unit {app} --model={ops_test.model.info.name} --attach-storage={data_storage_id} --attach-storage={archive_storage_id}"
+    add_unit_cmd = f"""add-unit {app} \
+                    --model={ops_test.model.info.name} \
+                    --attach-storage={data_storage_id} \
+                    --attach-storage={archive_storage_id} \
+                    --attach-storage={log_storage_id}
```

Comment:
Could you add a test where some storage volumes are added _post_ deployment? The `data` and `logs` ones ought to be interesting